moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -200,6 +205,16 @@ const ComponentForm = ({
             />
           </Grid>
         )}
+        <Grid item xs={12}>
+          <ControlledAutocomplete
+            id="tags"
+            label="Tags"
+            multiple
+            options={componentTagsOptions}
+            name="tags"
+            control={control}
+          />
+        </Grid>
```

Comment:
i think we should up the contrast on the highlights of the already-selected items. I feel like that's out of scope for this issue—i'll make a separate snackoo for it.

<img width="763" alt="Screenshot 2023-05-16 at 2 16 34 PM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/5611afd7-7d2d-4591-94f3-47fd6dfa75c6">